### PR TITLE
Board approval of activities

### PR DESCRIPTION
**Pull request by stefanmajoor**
Board can now see activities, and approve/disapprove them

Admin url: activity/admin/queue

### CODE REVIEW

File: module/Activity/config/module.config.php [hunk 1]
```diff
@@ -58,6 +58,56 @@
                                 'action' => 'create'
                             )
                         )
+                    ),
+                    'admin_queue' => array(
+                        'type' => 'Literal',
+                        'options' => array(
+                            'route' => '/admin/queue',
```
Comment (kokx) [inline module/Activity/config/module.config.php]: Admin routes currently have the structure `/admin/<module>/.....`. Please adhere to this in here. Also, it is better to make all these routes that start with `/admin/` childs of a general route.

File: module/Activity/src/Activity/Controller/AdminController.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                $activityService->approve($activity);
+                break;
+            case 'disapprove':
+                $activityService->disapprove($activity);
+                break;
+            case 'reset':
+                $activityService->reset($activity);
+                break;
+            default:
+                throw new \InvalidArgumentException('No sutch status ' . $status);
```
Comment (kokx) [inline module/Activity/src/Activity/Controller/AdminController.php]: The text here should be translated.

Comment (stefanmajoor) [inline module/Activity/src/Activity/Controller/AdminController.php]: this is not text that is returned to the user, so it does not have to be translated.

Also for the exception, I assume that will never reach the end user. and we are able to understand Dutch

File: module/Activity/config/module.config.php (hunk 1 above)
Comment (stefanmajoor) [inline module/Activity/config/module.config.php]: I will fix this tomorrow

**Comment by kokx** [comment]
Also, it has some merge conflicts. Please rebase on `master`.

**Comment by stefanmajoor** [comment]
I merged with master (after rebase failed), and the routes have been fixed

File: module/Activity/src/Activity/Mapper/Activity.php [hunk 3]
```diff
@@ -79,12 +94,27 @@ public function getUpcomingActivities($count = null)
      *
      * @return array
      */
-    public function getAllApproved()
+    public function getApprovedActivities()
```
Comment (kokx) [inline module/Activity/src/Activity/Mapper/Activity.php]: Because of DRY, is it not better to make one method for this, `findActivitiesByStatus`?

Comment (stefanmajoor) [inline module/Activity/src/Activity/Mapper/Activity.php]: Solved. Also only approved activities are shown in the upcomming  activities

**Pull request merged**